tests/cmdline/commands/test_presto.py

```diff
@@ -48,3 +48,24 @@ def test_presto(run_cli_command, with_broker, monkeypatch):
             assert profile.process_control_backend == 'core.rabbitmq'
         else:
             assert profile.process_control_backend is None
+
+
+@pytest.mark.usefixtures('empty_config')
+def test_presto_use_postgres(run_cli_command):
+    """Test the ``verdi presto`` with the ``-use-postgres`` flag."""
+    result = run_cli_command(verdi_presto, ['--non-interactive', '--use-postgres'])
+    assert 'Created new profile `presto`.' in result.output
+
+    with profile_context('presto', allow_switch=True) as profile:
+        assert profile.name == 'presto'
+        localhost = Computer.collection.get(label='localhost')
+        assert localhost.is_configured
+        assert profile.storage_backend == 'core.psql_dos'
```

Comment:
Could maybe run `verdi status` to verify the connection to DB?

Reply:
Did it by calling `Manager.get_profile_storage` which connects to the database and is more efficient than launching another CLI command